src/taco-cli/cli/remoteBuild/remoteBuildClientHelper.ts

```diff
@@ -148,8 +148,8 @@ class RemoteBuildClientHelper {
     }
 
     public static emulate(buildInfo: BuildInfo, serverSettings: Settings.IRemoteConnectionInfo, target: string): Q.Promise<BuildInfo> {
-        var buildUrlBase = Settings.getRemoteServerUrl(serverSettings) + "/build/" + buildInfo.buildNumber;
-        var httpSettings = { language: buildInfo.buildLang, agent: ConnectionSecurityHelper.getAgent(serverSettings) };
+        var buildUrlBase: string = Settings.getRemoteServerUrl(serverSettings) + "/build/" + buildInfo.buildNumber;
+        var httpSettings: { language: string; agent: Q.Promise<https.Agent> } = { language: buildInfo.buildLang, agent: ConnectionSecurityHelper.getAgent(serverSettings) };
```

Comment:
You are using this type 3 times. Consider declaring:
type HttpSettings = { language: string; agent: Q.Promise<https.Agent> };
and using
httpSettings: HttpSettings